src/components/bind-stores-mixin.coffee

```diff
@@ -0,0 +1,28 @@
+_ = require 'underscore'
+
+module.exports =
+
+  # can modify which event you want to bind on as needed.
+  _defaultEvent: 'change'
+
+  _defaultUpdate: ->
+    @forceUpdate()
+
+  _addListener: (bind) ->
+    bind.store.on(bind.listenTo or @_defaultEvent, bind.callback or @_defaultUpdate)
+
+  _removeListener:  (bind) ->
+    bind.store.off(bind.listenTo or @_defaultEvent, bind.callback or @_defaultUpdate)
+
+  _addListeners: ->
+    bindEvents = @_getBindEvents?()
```

Comment:
It seems `_getBindEvents` must be defined on the `Component` that uses this mixin; if so, then should this not have the underscore (since it is not a private method)?